Source/WTF/wtf/ArgumentCoder.h

```diff
@@ -92,9 +92,9 @@ struct ArgumentCoder<bool> {
     template<typename Decoder>
     static std::optional<bool> decode(Decoder& decoder)
```

Comment:
Note (not related to your patch per se):
this !!bool pattern seems futile, as other bools inside trivially constructible structs are being passed as data.
so individual bools are "sanitized" as 1, but bools inside data are not. Which makes this one seem pointless.

Reply:
I'll apply this recommendation later.